src/common/console_service.h

```diff
@@ -19,6 +19,9 @@
 ===========================================================================
 */
 
+#ifndef _CONSOLE_SERVICE_H_
+#define _CONSOLE_SERVICE_H_
```

Comment:
No need for header guards like this, #pragma once does the same thing, it's just not part of the standard (or something)

Reply:
Was failing on GCC10 without these